src/lib/data.ts

```diff
@@ -675,5 +681,4 @@ function load_from_local_storage_and_sync() {
     }
 }
 
-load_from_local_storage_and_sync();
```

Comment:
I want to say this was here for a reason, likely to solve a race condition. I probably should have put a comment..

Reply:
I have logged the heck out of it, I think that actually it is not needed, but it doesn't hurt because it does nothing.

In fact, because it does nothing, the race that I thought I found before does not exist: although this is called here, it is short-circuited by loaded_user_id being set the first time:

```
function load_from_local_storage_and_sync() {
    const user = store["config.user"];
   //...
    if (loaded_user_id === user.id) {
        return;
    }
```

This results in the first `remote_sync` not happening till rather later when the socket connects.